CONTRIBUTING.md

```diff
@@ -129,7 +129,7 @@ Other notes:
   understand why it is difficult and are confident you can resolve the
   issue correctly and completely.  Issues without one of these labels
   are fair game if Tim has written a clear technical design proposal
-  in the issue, or it is a bug that you can reproduce and you are
+  in the issue, or it is a bug that you can reproduce, and you are
```

Comment:
I think this change is wrong; the `and is meant to only modify the second case of you can reproduce it.

Reply:
Okay, removed this change in the latest push.